src/rust/bitbox02/src/ui/types.rs

```diff
@@ -86,3 +89,35 @@ impl<'a> ConfirmParams<'a> {
         }
     }
 }
+
+pub struct SelectWordCb<'a>(pub Box<dyn FnMut(u8) + 'a>);
```

Comment:
Why don't you just use type aliases? https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=be5c51bd5ebc72a6afee5976551ccebe

Reply:
:thinking: i thought I tried but didn't work, but obviously it does... Cool, let me try again.

Reply:
Done! Thanks.